src/main/java/org/zalando/nakadi/util/FeatureToggleService.java

```diff
@@ -26,7 +26,6 @@
         DISABLE_EVENT_TYPE_DELETION("disable_event_type_deletion"),
         HIGH_LEVEL_API("high_level_api"),
         CHECK_APPLICATION_LEVEL_PERMISSIONS("check_application_level_permissions"),
-        CHECK_PARTITIONS_KEYS("check_partitions_keys"),
```

Comment:
@v-stepanov it's not. But we cannot release this feature without enabling it first. More details here https://github.com/zalando/nakadi/pull/494/files#r91092420

Do you remember why is there a feature toggle for it?